src/main.ts

```diff
@@ -30,9 +32,81 @@ const getMongoDBClient = async (): Promise<NoSQLWrapper> => {
         const result = await db.collection('users').find({}).toArray();
         return result;
     }
+
+    const FindOneUser = async (email:string, password: string): Promise<any> => {
+        const result = await db.collection('users').findOne({email,password});
+        return result;
+    }
+
+    const CreateArticulo = async (articulo: any): Promise<any> => {
```

Comment:
change the name of this function something like CrateArticle don't mix two languages